tensorflow_datasets/core/features/image_feature_test.py

```diff
@@ -109,6 +109,22 @@ def test_image_shaped(self):
         ],
     )
 
+  def  test_image_dtype(self):
+      img = randint(256 , size=(128,128,3) ,dtype=np.uint8)
+      
+      # Image with statically defined dtype   
+      self.assertFeature(
+        feature=features_lib.Image( dtype=tf.uint16),
+        shape=(None, None, 3),
+        dtype=tf.uint8,
```

Comment:
Why is output `dtype=tf.uint8` if the `Image` is set to `tf.uint16` ?
You should also modify the `get_tensor_info()`

Reply:
Forgot to change dtype in `get_tensor_info()`  and  the output dtype also, now modified it! 
Thank you!!